molgenis-api-metadata/src/main/java/org/molgenis/api/metadata/v3/job/SerializableAttribute.java

```diff
@@ -16,6 +16,8 @@
 
   abstract String getName();
 
+  abstract String getEntityTypeId();
```

Comment:
What is the purpose of this method since the attribute is persisted as part of the entity type that it belong to?